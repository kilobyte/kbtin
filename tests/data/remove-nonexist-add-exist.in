#anti x
#unanti y
#anti x
#anti
#sub x 1
#unsub y
#sub x 2
#sub
#high green x
#unhi y
#high red x
#hi
#act x 1
#unact y
#act x 2
#act
#prom x 1
#unprom y
#prom x 2
#prom
#show **********
#unanti x
#anti
#unsub x
#sub
#unhi x
#hi
#unact x
#act
#unprom x
#prom
#show **********
