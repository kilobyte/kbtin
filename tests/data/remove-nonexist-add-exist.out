#THESE ANTISUBSTITUTES HAS BEEN DEFINED:
{x}
#THESE SUBSTITUTES HAVE BEEN DEFINED:
{x}={2}
#THESE HIGHLIGHTS HAVE BEEN DEFINED:
{x}={red}
#Defined actions:
{x}={2} @ {5}
#Defined promptactions:
{x}={2} @ {5}
**********
#THESE ANTISUBSTITUTES HAS BEEN DEFINED:
#NO SUBSTITUTES HAVE BEEN DEFINED.
#THESE HIGHLIGHTS HAVE BEEN DEFINED:
#Defined actions:
#Defined promptactions:
**********
